ix-dev/community/esphome/questions.yaml

```
@@ -0,0 +1,386 @@
+groups:
+  - name: esphome Configuration
```

```suggestion
  - name: ESPHome Configuration
```
Please rename everywhere here

---

updated